### ISSUE TITLE
Creating contributing guidelines

### ISSUE THREAD

**Issue by vladimirkosolapov** [morning4coffe-dev/storylines#8]
It would be a good idea to create CONTRIBUTING.md in repository for the convenience of contributors. I'd like to help with internationalization and localization, but I expect a detailed description of this process in the context of this app.

As an example of such a file in the UWP app repository, I can refer to [DevToys](https://github.com/veler/DevToys#contribute).

**Comment by morning4coffe-dev** [comment]
Thank you for your great suggestion! I hope to have some free time tomorrow, so I will look into it.

**Comment by morning4coffe-dev** [comment]
I created the [CONTRIBUTING to Storylines](https://github.com/morning4coffe-dev/storylines/blob/main/CONTRIBUTING.md) document with some guidelines on how to run and build Storylines and what to make sure and what to avoid during the Internationalization and localization process. Expect an update with more description on how to add a new translation into the project soon, but the process should be similar to the DevToys.

**Comment by vladimirkosolapov** [comment]
This is certainly good, but the best explanation is one that doesn't involve working with Visual Studio to translate at all. Is it possible to translate Storylines just by editing the files manually? If so, that's better.

**Comment by morning4coffe-dev** [comment]
To be honest I am not quite sure. There is a way to not use the MAT editor and just open the .xlf file in some text editor and edit the code, but I don't know about a way to generate those files without the VS extension. And if there is a way, then you would still need it to build the app with the new translation for testing. But could be interesting to try.

**Comment by morning4coffe-dev** [comment]
Thank you again for this great suggestion. 
Yesterday, I updated the [CONTRIBUTING to Storylines document](https://github.com/morning4coffe-dev/storylines/blob/70ca622efa7209a122a45403b15ed86699f7b01e/CONTRIBUTING.md) with the 0.6.8 code update I have been working on. It now contains all the important things to Build and Run Storylines from the source and to add/update translations even without the MAT editor. 
Unfortunately, I did not find a way to avoid using MAT extension, but that said I am currently starting to work on a #14 and I am currently experimenting with some other possibilities.
